Optik/Classes/AlbumViewController.swift

```diff
@@ -84,12 +85,14 @@ internal final class AlbumViewController: UIViewController {
         self.dismissButtonPosition = dismissButtonPosition
         
         pageViewController = UIPageViewController(transitionStyle: .scroll,
-                                                  navigationOrientation: .horizontal,
+                                                  navigationOrientation: .vertical,
                                                   options: [UIPageViewControllerOptionInterPageSpacingKey : Constants.SpacingBetweenImages])
-
+        
+        pageControl = UIPageControl()
```

Comment:
All the variables are initialized in the init method. 
The setup methods is only to configure the objects. 

Should I move to setup the page control initializer?